chia/_tests/core/data_layer/test_data_rpc.py

```diff
@@ -2288,7 +2288,7 @@ async def test_wallet_log_in_changes_active_fingerprint(
 
     mnemonic = create_mnemonic()
     assert wallet_rpc_api.service.local_keychain is not None
-    private_key = wallet_rpc_api.service.local_keychain.add_key(mnemonic_or_pk=mnemonic)
+    private_key, _ = wallet_rpc_api.service.local_keychain.add_key(mnemonic_or_pk=mnemonic)
```

Comment:
ugly tuple mucking.....

Reply:
Is this just pointing out that you think the function shouldn't return a tuple or would you rather `(...)[0]` or something?

Reply:
I'm not sure , but this change made everything that calls this now have to do `key, _ = ` - and rarely are you using the key type element from the tuple.
maybe it's a stepping stone to using a proper class here though

Reply:
Most of the instances where we use the `, _` is in test code, there's three instances in production code and there could be an argument to use the types there.  Like "added key of type blah with fingerprint blah".  I'll wait until the discussion of whether or not this should be returning a tuple is resolved and then I will add that in if so.

Reply:
alrighty